dali/operators/numba_function/numba_func.cu

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include "dali/operators/numba_function/numba_func.h"
+
+namespace dali {
+
+vector<ssize_t> calc_sizes(DALIDataType type, int ndim, uint64_t shape_ptr) {
+  ssize_t item_size = TypeTable::GetTypeInfo(type).size();
+
+  vector<ssize_t> args;
+  args.push_back(item_size);
+
+  size_t nitems = 1;
+  uint64_t* shapes = (uint64_t*)shape_ptr;
```

Comment:
I think linter may not like this C style cast, use `static_cast<uint64_t*>()`.
Check other places as well.

Reply:
Done